GitUI/GitUICommands.cs

```diff
@@ -1505,9 +1505,6 @@ private void RunCommandBasedOnArgument(IReadOnlyList<string> args, IReadOnlyDict
                 case "formatpatch":
                     StartFormatPatchDialog();
                     return;
-                case "gitbash":
```

Comment:
Yes, I did remove it, as I removed `GitModule.RunBash()` in 14690b0 as it duplicates the functionality of running bash that we have in `FormBrowse` in a way that is excessive.

Reply:
Ok. So I pushed the fixup...

Reply:
I think I got why you removed `GitModule.RunBash`, @RussKie. But dropping a CLI in this PR feels like a submarine.
It breaks the usecase: `bash` and e.g. `grep` *not* in the `PATH`, but `gitex.cmd` in the `PATH`.
So `gitex gitbash` should open a bash.

Reply:
I mulled over it, and you're probably right - it's a "dot" release, and this could be a breaking behaviour... we can drop  14690b0 from this PR, but I intent do it in the master.
 

> It breaks the usecase: `bash` and e.g. `grep` _not_ in the `PATH`, but `gitex.cmd` in the `PATH`.
> So `gitex gitbash` should open a bash.

If bash is not in the path, then we can't open it. Trying to open bash via the app feels like a wrong.

Reply:
This removal of the CLI command "gitbash" must be mentioned in the change log.